recipes/openscenegraph/all/conanfile.py

```diff
@@ -68,6 +69,7 @@ class OpenSceneGraphConanFile(ConanFile):
         "with_png": True,
         "with_tiff": True,
         "with_zlib": True,
+        "opengl_profile": "GL2",
```

Comment:
Is it the default value on the upstream?

Reply:
Yes, that's it. You can have a look exactly here, GL2 is assumed by default:
https://github.com/openscenegraph/OpenSceneGraph/blob/master/CMakeLists.txt#L491

Reply:
Thank you for checking!!